modules/apps/portal-search/portal-search-web/src/main/resources/META-INF/resources/js/components/search_bar_configuration_suggestions/contributor_input_set_item/inputs/SXPBlueprintSelectorInput.js

```diff
@@ -115,7 +115,9 @@ function SXPBlueprintSelectorInput({
 					observer={observer}
 					onClose={onClose}
 					onSubmit={_handleSubmit}
-					selectedExternalReferenceCode={sxpBlueprintExternalReferenceCode || ''}
+					selectedExternalReferenceCode={
```

Comment:
Brian may not like SFs to be separate commits, but this is not much so could be fine